src/Markdig.Wpf/Themes/generic.xaml

```diff
@@ -61,4 +62,16 @@
     <Setter Property="Stretch" Value="Fill" />
     <Setter Property="Stroke" Value="Black" />
   </Style>
+    
+  <!-- MarkdownViewer Template-->  
+  <Style TargetType="markdig:MarkdownViewer">
+    <Setter Property="Template">
+      <Setter.Value>
+        <ControlTemplate TargetType="markdig:MarkdownViewer">
+          <FlowDocumentScrollViewer Document="{TemplateBinding Document}"
+                                    ScrollViewer.VerticalScrollBarVisibility="Auto"/>
```

Comment:
I don't remember if there is a way to have the scrollbar visibility properties also templated so that a user of `MarkdownViewer` can set the `ScrollViewer.VerticalScrollBarVisibility` or `ScrollViewer.HorizontalScrollBarVisibility` and it will apply correctly on the inner `FlowDocumentScrollViewer `? That would be nice I think.